README.md

```diff
@@ -5,49 +5,127 @@
+</a>
+<a href="https://sonarcloud.io/summary/new_code?id=jellyfin_jellyfin-vue">
+<img alt="Reliability Rating"src="https://sonarcloud.io/api/project_badges/measure?project=jellyfin_jellyfin-vue&metric=reliability_rating"/>
+</a>
+<a href="https://sonarcloud.io/summary/new_code?id=jellyfin_jellyfin-vue">
+<img alt="Vulnerabilities"src="https://sonarcloud.io/api/project_badges/measure?project=jellyfin_jellyfin-vue&metric=vulnerabilities"/>
+</a>
+<a href="https://sonarcloud.io/summary/new_code?id=jellyfin_jellyfin-vue">
+<img alt="Code Smells"src="https://sonarcloud.io/api/project_badges/measure?project=jellyfin_jellyfin-vue&metric=code_smells"/>
+</a>
+<a href="https://sonarcloud.io/summary/new_code?id=jellyfin_jellyfin-vue">
+<img alt="Technical debt"src="https://sonarcloud.io/api/project_badges/measure?project=jellyfin_jellyfin-vue&metric=sqale_index"/>
+</a>
+<a href="https://sonarcloud.io/summary/new_code?id=jellyfin_jellyfin-vue">
+<img alt="Quality gate"src="https://sonarcloud.io/api/project_badges/measure?project=jellyfin_jellyfin-vue&metric=alert_status"/>
+</a>
+<a href="https://sonarcloud.io/summary/new_code?id=jellyfin_jellyfin-vue">
+<img alt="Duplicated lines"src="https://sonarcloud.io/api/project_badges/measure?project=jellyfin_jellyfin-vue&metric=duplicated_lines_density"/>
+</a>
+<a href="https://sonarcloud.io/summary/new_code?id=jellyfin_jellyfin-vue">
+<img alt="Bugs"src="https://sonarcloud.io/api/project_badges/measure?project=jellyfin_jellyfin-vue&metric=bugs"/>
+</a>
+<a href="https://sonarcloud.io/summary/new_code?id=jellyfin_jellyfin-vue">
+<img alt="Security"src="https://sonarcloud.io/api/project_badges/measure?project=jellyfin_jellyfin-vue&metric=security_rating"/>
+</a>
+<a href="https://sonarcloud.io/summary/new_code?id=jellyfin_jellyfin-vue">
+<img alt="Total lines"src="https://sonarcloud.io/api/project_badges/measure?project=jellyfin_jellyfin-vue&metric=ncloc"/>
+</a>
 </p>
 
-This is an experimental web client for Jellyfin based on Vue.js. We welcome all contributions and pull requests! If you have a larger feature in mind please open an issue so we can discuss the implementation before you start.
+---
 
-<p align="center">
-    <a href="https://jf-vue.pages.dev/">View demo</a>
-</p>
+This is an alternative client for Jellyfin based on Vue.js. It might not be feature complete and it's constantly evolving.
+
+# For end users 👨‍👩‍👧‍👦
+
+## [Hosted instance 🌍](https://jf-vue.pages.dev/)
+
+Our hosted instance points to a version built from the current `master` branch. Hosted by CloudFlare Pages.
+
+## Docker Container 📦
+
+You can run the Docker container with the following command, replacing 8080 with the port of your choice:
+
+```bash
+docker run -d -p 8080:80 ghcr.io/jellyfin/jellyfin-vue
+```
+
+You can check [Packages](https://github.com/jellyfin/jellyfin-vue/pkgs/container/jellyfin-vue) (recommended) or [DockerHub](https://hub.docker.com/r/jellyfin/jellyfin-vue) for all the tagged images.
```

Comment:
```suggestion
You can check [GHCR](https://github.com/jellyfin/jellyfin-vue/pkgs/container/jellyfin-vue) (recommended) or [DockerHub](https://hub.docker.com/r/jellyfin/jellyfin-vue) for all the tagged images.
```

I've never seen the GH repo being called that way, maybe GHCR isn't the best but I don't like Packages